packages/react-windows.1.2.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "1.2"
+maintainer: "Pierre Boutillier <pierre.boutillier@laposte.net>"
+homepage: "http://erratique.ch/software/react"
+authors: ["Daniel Bünzli <daniel.buenzl i@erratique.ch>"]
+doc: "http://erratique.ch/software/react/doc/React"
+#dev-repo: "http://erratique.ch/repos/react.git"
+#bug-reports: "https://github.com/dbuenzli/react/issues"
+tags: [ "reactive" "declarative" "signal" "event" "frp" "org:erratique" ]
+license: "BSD3"
+depends: [
+  "ocaml-windows"
+  "ocamlfind"
+  "ocamlbuild" {build}
+]
+build:
+[
+  [ "ocaml" "pkg/git.ml" ]
+  [ "env" "OCAMLFIND_TOOLCHAIN=windows"
+    "ocamlbuild" "-use-ocamlfind" "-pkg" "compiler-libs.toplevel"
+      "src/react.cma" "src/react_top.cma" "src/react.cmxa" "src/react_top.cmxa"
+      "src/react.cmxs" "src/react_top.cmxs" "pkg/META"
+ ]
+]
+install: [["ocamlfind" "-toolchain" "windows" "install" "react"
+           "_build/src/react.cmxs" "_build/src/react.cmxa" "_build/src/react.a"
+           "_build/src/react.cmx" "_build/src/react.cmi" "_build/src/react.mli"
+           "_build/src/react.cma" "_build/src/react_top.cma"
+	   "_build/src/react_top.cmxa" "_build/src/react_top.cmxs"
+	   "_build/src/react_top.a" "_build/pkg/META" ]]
```

Comment:
Please don't do this. See the section on topkg in https://github.com/whitequark/opam-cross-windows#porting-packages.